src/Integration/Binding/ProjectBindingOperation.cs

```diff
@@ -51,7 +51,10 @@ internal partial class ProjectBindingOperation : IBindingOperation
         {
         }
 
-        internal ProjectBindingOperation(IServiceProvider serviceProvider, Project project, ISolutionBindingConfigFileStore configFileStore, IFileSystem fileSystem)
+        internal ProjectBindingOperation(IServiceProvider serviceProvider, 
```

Comment:
Most of the code in SLVS doesn't separate constructor arguments onto separate lines. Did you make this change just for stylistic reasons? If so, can we have an offline discussion about how we are going to handle unnecessary stylistic changes please?

Reply:
I saw a lot of code in SLVS that does separate, so I agree, we should decide on a convention.
I'll schedule a meeting? we can discuss other things as well, like tests naming, etc.